### DISCUSSION TITLE
Next.js project is not crawlable by facebook debug

### DISCUSSION THREAD

**Discussion by imran1992** [vercel/next.js#14438] [Help] [answered]
Next.js project is working on browser as excepted. But Facebook debug to only read data from _app.js but not from other pages.
I am putting headers for Seo purpose in these pages But as facebook doesn't crawl through so its not working.
I am checking url at https://developers.facebook.com/tools/debug/
- next.js v9.3.6
- node v14.4.0

**Comment by oshanfernando** [comment]
You mean you cannot see the preview and title ? If that's the case check if you have added the Open graph meta tags properly

**Comment by imran1992** [comment, marked as answer]
Its is resolved. I was using redux-persist. Because of that serverside is not render other pages. But client is rending.
I just remove redux-persist for server using process.browser global veriable.
